index.html

```diff
@@ -1,53 +1,179 @@
+                <div class="point"></div>
+                <p>2015</p>
+            </div>
+
+            <p id="para1">A daily selection of privately <br>personalized roads; no accounts or <br>sign-up required.
+            </p>
+            <ul>
+                <li id="html"><a href="#"> html</a></li>
+                <li id="css"><a href="#">css</a></li>
+                <li id="javaScript"><a href="#">javaScript</a></li>
+            </ul>
+            <div class="to-project"><a href="#">See Project</a></div>
+        </div>
+        <div class="container container4">
+            <div class="Card1"><img src="images/img4.png" alt="" class="card1"></div>
+            <div>
+                <h1>Multi-Post <br>Stories</h1>
+            </div>
+            <div class="lister">
+                <p><strong>CANOPY</strong></p>
+                <div class="point"></div>
+                <P>Back End Dev</P>
+                <div class="point"></div>
+                <p>2015</p>
+            </div>
+
+            <p id="para1">A daily selection of privately <br>personalized roads; no accounts or <br>sign-up required.
+            </p>
+            <ul>
+                <li id="html"><a href="#"> html</a></li>
+                <li id="css"><a href="#">css</a></li>
+                <li id="javaScript"><a href="#">javaScript</a></li>
+            </ul>
+            <div class="to-project"><a href="#">See Project</a></div>
+        </div>
+        <section class="About-section">
+            <div class="about-title">About <br> Myself</div>
+            <p class="para2">Hello I'm a software developer I can help you build a product, feature or website
+                look through some of my work and experience! if you like what you see and havd a Project
+                you need coded, don't hesitate to contact me.
+                <div class="connect">LET'S CONNECT</div>
+            </p>
+            <ul>
+                <li><a href=""><img src="./images/github.png" /></a></li>
+                <li><a href=""><img src="./images/linkedin.jpeg" /></a></li>
+                <li><a href=""><img src="./images/sign.png" /></a></li>
+                <li><a href=""><img src="./images/twiter.png" /></a></li>
+                <li><a href=""> <img src="./images/Muliti.png" /></a></li>
+            </ul>
+
+            <div class="get">Get my resume</div>
+            <ul>
+                <li>
+                    <div class="lang-dropper">Langguages</div>
+                </li>
+            </ul>
+            <div class="lang"><img src="images/js.png" alt="">javaScript</div>
+            <div class="lang"><img src="images/html.png" alt="">HTML</div>
+            <div class="lang"><img src="images/CSS.png" alt="">CSS</div>
+            <ul class="footer-list">
+                <li></li>
+                <div class="frame">Frameworks</div>
+                </li>
+                <li>
+                    <div class="frame">Skills</div>
+                </li>
```

Comment:
- [ ] You have done well to implement the hover button effect in other buttons, Could you kindly do the same to the Get Resume Button. Having the right hover effect helps improve the experience of your product's users
- [ ] The languages and their icons are a bit cramped together, let us add a little space as shown in the Figma template attached.
- [ ] Finally, let us add the icons as shown in the Figma template. Icons are a very important piece of website designs and therefore a very important skill to learn. 

![Screenshot from 2022-02-17 09-44-34](https://user-images.githubusercontent.com/50773868/154439104-e6fcd7dd-8a47-45db-8154-cefa30f24fd9.png)

Reply:
all fonts match with the figma design in my css file, please kindly verify before comment.